main.go

```diff
@@ -56,8 +58,9 @@ func main() {
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
 	ctrlOptions := ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: "0", // disable metrics
+		Scheme:                 scheme,
+		MetricsBindAddress:     "0",     // disable metrics
```

Comment:
related ?

Reply:
is the golang formating aligning the code, this is fine, it should follow golang standard fmt.

Reply:
ah misread